src/PooltogetherGovernor.sol

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.18;
+
+import {
+  Governor, GovernorCountingFractional
+} from "flexible-voting/src/GovernorCountingFractional.sol";
+import {
+  ERC20VotesComp,
+  GovernorVotesComp
+} from "@openzeppelin/contracts/governance/extensions/GovernorVotesComp.sol";
+import {IGovernor} from "@openzeppelin/contracts/governance/IGovernor.sol";
+import {
+  GovernorTimelockCompound,
+  ICompoundTimelock
+} from "@openzeppelin/contracts/governance/extensions/GovernorTimelockCompound.sol";
+import {GovernorSettings} from "@openzeppelin/contracts/governance/extensions/GovernorSettings.sol";
+
+/// @notice The upgraded Pooltogether Governor: Bravo compatible and built with OpenZeppelin.
+contract PooltogetherGovernor is
+  GovernorCountingFractional,
+  GovernorVotesComp,
+  GovernorTimelockCompound,
+  GovernorSettings
+{
+  /// @notice The address of the POOL token on Ethereum mainnet from which this Governor derives
+  /// delegated voting weight.
+  ERC20VotesComp private constant POOL_TOKEN =
+    ERC20VotesComp(0x0cEC1A9154Ff802e7934Fc916Ed7Ca50bDE6844e);
```

Comment:
How concerned are we if some methods are not present on the token contract? One example would be `getVotes` is missing, but there is a replacement `getCurrentVotes`.

Reply:
The big thing we'll need to understand is whether the existing contract ABI's play nicely with what the OZ extensions are expecting, or if they are breaking changes that will require us to modify the OZ contracts in some way. I can help dig into this—it should also become apparent once we start writing tests, which would fail if the signatures didn't work.

Reply:
Ah okay, let's cross that bridge once we get there then. #3 for reference

Reply:
I think we should investigate this deeper as a standalone task, but agree it can wait until after this PR is merged. https://github.com/ScopeLift/pooltogether-governor-upgrade/issues/5